molgenis-one-click-importer/src/main/java/org/molgenis/oneclickimporter/service/EntityService.java

```diff
@@ -5,5 +5,11 @@
 
 public interface EntityService
 {
-	EntityType createEntity(DataCollection dataCollection);
+	/**
+	 * Create an {@link EntityType} from a {@link DataCollection}
+	 *
+	 * @param dataCollection
+	 * @return the ID of the newly created {@link EntityType}
+	 */
+	String createEntityType(DataCollection dataCollection);
```

Comment:
i would expect to get the entity, instead of just the id, the entity is nice and typed, the string is not :(, this sort of thing leads to spaghetti in my experience